### PR TITLE
Make Ignore.pathIsIgnored async

### PR DESCRIPTION
**Pull request by joshaber**
Since it’s potentially doing IO and touching global state, it should be async.

### CODE REVIEW

File: generate/input/descriptor.json [hunk 1]
```diff
@@ -949,9 +949,11 @@
         "git_ignore_path_is_ignored": {
           "args": {
             "ignored": {
+              "shouldAlloc": true,
               "isReturn": true
             }
-          }
+          },
+          "isAsync": true
```
Comment (nathansobo) [inline generate/input/descriptor.json]: Wow it's pretty cool that this is all it took.

Comment (joshaber) [inline generate/input/descriptor.json]: 💥 👶

Comment (nathansobo) [inline generate/input/descriptor.json]: Where'd this baby come from!? Now I gotta take care of it.

Comment (tbranyen) [inline generate/input/descriptor.json]: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  :tada: &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; :tada:
&nbsp;&nbsp;&nbsp;  :tada:  yeah, it's pretty ~cool~&nbsp; &nbsp; &nbsp; &nbsp;  :tada:
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  :tada: &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; :tada:
&nbsp;&nbsp;&nbsp;&nbsp; :tada:; &nbsp; &nbsp; :tada:
:penguin:

**Comment by johnhaley81** [comment]
image

lol

**Comment by joshaber** [comment]
This should be ready to merge AFAIK.

**Comment by johnhaley81** [comment]
LGTM. Thanks @joshaber

**Comment by joshaber** [comment]
🙇

**Pull request merged**